kubernetes/yamls/development/api-server/api-deployment.yml

```diff
@@ -18,13 +18,8 @@ spec:
           httpGet:
             path: /api/status.json
             port: 80
-          initialDelaySeconds: 30
-          timeoutSeconds: 3
-        readinessProbe:
-          httpGet:
-            path: /api/status.json
-            port: 80
-          initialDelaySeconds: 30
+          initialDelaySeconds: 90
+          periodSeconds: 300
           timeoutSeconds: 3
```

Comment:
So here is the issue - Elasticsearch is not present at `127.0.0.1`. It is there are `elasticsearch.elasticsearch:9300`. Please use proper image to set things up.